echopype/echodata/combine.py

```diff
@@ -180,44 +225,24 @@ def combine_echodata(echodatas: List[EchoData], combine_attrs="override") -> Ech
                     # TODO: investigate further why we need to do .astype("<U50")
                     combined_group["channel"] = combined_group["channel"].astype("<U50")
 
-            if sonar_model in ("EK60", "EK80"):
-                if "ping_time" in combined_group and exist_reversed_time(
-                    combined_group, "ping_time"
-                ):
-                    if old_ping_time is None:
-                        warnings.warn(
-                            f"{sonar_model} ping_time reversal detected; the ping times will be corrected"  # noqa
-                            " (see https://github.com/OSOceanAcoustics/echopype/pull/297)"
-                        )
-                        old_ping_time = combined_group["ping_time"]
-                        coerce_increasing_time(combined_group, time_name="ping_time")
-                        new_ping_time = combined_group["ping_time"]
-                    else:
-                        combined_group["ping_time"] = new_ping_time
-                if "time1" in combined_group and exist_reversed_time(combined_group, "time1"):
-                    if group != "nmea":
-                        if old_time1 is None:
-                            warnings.warn(
-                                f"{sonar_model} time1 reversal detected; the location times will be corrected"  # noqa
-                                " (see https://github.com/OSOceanAcoustics/echopype/pull/297)"
-                            )
-                            old_time1 = combined_group["time1"]
-                            coerce_increasing_time(combined_group, time_name="time1")
-                            new_time1 = combined_group["time1"]
-                        else:
-                            combined_group["time1"] = new_time1
-            if sonar_model == "EK80":
-                if "time2" in combined_group and exist_reversed_time(combined_group, "time2"):
-                    if old_time2 is None:
-                        warnings.warn(
-                            f"{sonar_model} time2 reversal detected; the mru times will be corrected"  # noqa
-                            " (see https://github.com/OSOceanAcoustics/echopype/pull/297)"
-                        )
-                        old_time2 = combined_group["time2"]
-                        coerce_increasing_time(combined_group, time_name="time2")
-                        new_time2 = combined_group["time2"]
-                    else:
-                        combined_group["time2"] = new_time2
+            if sonar_model in ("EK60", "EK80", "AZFP"):
```

Comment:
How about just excluding AD2CP `sonar_model` == "AD2CP"? That would have the added advantage of including ES70, ES80 and EA640, which I assume should be included too.
```suggestion
            if sonar_model != "AD2CP":
```